### PR TITLE
Add callout for anyone using Traefik to manage commercial applications

### PR DESCRIPTION
**Pull request by tomatokoolaid**
### What does this PR do?

Adds a callout at the end for anyone using Traefik to manage commercial applications who might need more advanced networking and security functionality.

### Motivation

Wanted to provide an avenue for users who need additional functionality beyond what Traefik Proxy and Hub provide.

### More

- [x] Added/updated documentation

### CODE REVIEW

File: docs/content/providers/kubernetes-ingress.md [hunk 1]
```diff
@@ -501,3 +501,7 @@ providers:
 
 To learn more about the various aspects of the Ingress specification that Traefik supports,
 many examples of Ingresses definitions are located in the test [examples](https://github.com/traefik/traefik/tree/v2.8/pkg/provider/kubernetes/ingress/fixtures) of the Traefik repository.
+
+!!! question "Using Traefik for Business Applications?"
+
+    If you're using Traefik for commercial applications, consider the [Enterprise Edition](https://traefik.io/traefik-enterprise/) of Traefik as your [Kubernetes Ingress](https://traefik.io/solutions/kubernetes-ingress/) or your [API gateway](https://traefik.io/solutions/api-gateway/) to enable access management, distributed Let's Encrypt, and other advanced capabilties. Learn more in [this 15-minute technical walkthrough](https://info.traefik.io/watch-traefikee-demo).
```
Comment (tomMoulard) [inline docs/content/providers/kubernetes-ingress.md:507]:
```suggestion
    If you're using Traefik for commercial applications,
    consider the [Enterprise Edition](https://traefik.io/traefik-enterprise/) of Traefik as your [Kubernetes Ingress](https://traefik.io/solutions/kubernetes-ingress/)
    or your [API gateway](https://traefik.io/solutions/api-gateway/)
    to enable access management, distributed Let's Encrypt, and other advanced capabilties.
    Learn more in [this 15-minute technical walkthrough](https://info.traefik.io/watch-traefikee-demo).
```

File: docs/content/routing/entrypoints.md [hunk 2]
```diff
@@ -967,3 +967,7 @@ entryPoints:
 entrypoints.foo.address=:8000/udp
 entrypoints.foo.udp.timeout=10s
 ```
+
+!!! question "Using Traefik for Business Applications?"
+
+    If you're using Traefik for commercial applications, consider the [Enterprise Edition](https://traefik.io/traefik-enterprise/) of Traefik as your [Kubernetes Ingress](https://traefik.io/solutions/kubernetes-ingress/), your [Docker Swarm Load Balancer](https://traefik.io/solutions/docker-swarm-ingress/), or your [API gateway](https://traefik.io/solutions/api-gateway/) to enable access management, distributed Let's Encrypt, and other advanced capabilties. Learn more in [this 15-minute technical walkthrough](https://info.traefik.io/watch-traefikee-demo).
```
Comment (tomMoulard) [inline docs/content/routing/entrypoints.md:973]:
```suggestion
    If you're using Traefik for commercial applications,
    consider the [Enterprise Edition](https://traefik.io/traefik-enterprise/) of Traefik as your [Kubernetes Ingress](https://traefik.io/solutions/kubernetes-ingress/),
    your [Docker Swarm Load Balancer](https://traefik.io/solutions/docker-swarm-ingress/),
    or your [API gateway](https://traefik.io/solutions/api-gateway/)
    to enable access management, distributed Let's Encrypt, and other advanced capabilties.
    Learn more in [this 15-minute technical walkthrough](https://info.traefik.io/watch-traefikee-demo).
```

File: docs/content/routing/services/index.md [hunk 3]
```diff
@@ -1645,3 +1645,7 @@ udp:
       [[udp.services.appv2.loadBalancer.servers]]
         address = "private-ip-server-2:8080/"
 ```
+
+!!! question "Using Traefik for Business Applications?"
+
+    If you're using Traefik for commercial applications, consider the [Enterprise Edition](https://traefik.io/traefik-enterprise/) of Traefik as your [Kubernetes Ingress](https://traefik.io/solutions/kubernetes-ingress/), your [Docker Swarm Load Balancer](https://traefik.io/solutions/docker-swarm-ingress/), or your [API gateway](https://traefik.io/solutions/api-gateway/) to enable access management, distributed Let's Encrypt, and other advanced capabilties. Learn more in [this 15-minute technical walkthrough](https://info.traefik.io/watch-traefikee-demo).
```
Comment (tomMoulard) [inline docs/content/routing/services/index.md:1651]:
```suggestion
    If you're using Traefik for commercial applications,
    consider the [Enterprise Edition](https://traefik.io/traefik-enterprise/) of Traefik as your [Kubernetes Ingress](https://traefik.io/solutions/kubernetes-ingress/),
    your [Docker Swarm Load Balancer](https://traefik.io/solutions/docker-swarm-ingress/),
    or your [API gateway](https://traefik.io/solutions/api-gateway/)
    to enable access management, distributed Let's Encrypt, and other advanced capabilties.
    Learn more in [this 15-minute technical walkthrough](https://info.traefik.io/watch-traefikee-demo).
```

File: docs/content/providers/kubernetes-ingress.md (hunk 1 above)
Comment (tomatokoolaid) [reply docs/content/providers/kubernetes-ingress.md]: Thanks, Tom. I'll try to make them like in the future.

File: docs/content/getting-started/configuration-overview.md [hunk 4]
```diff
@@ -93,3 +93,12 @@ All available environment variables can be found [here](../reference/static-conf
...
 You can browse the available features in the menu, the [providers](../providers/overview.md), or the [routing section](../routing/overview.md) to see them in action.
+
+!!! question "Using Traefik for Business Applications?"
+
+If you're using Traefik for commercial applications, consider the [Enterprise Edition](https://traefik.io/traefik-enterprise/). You can use it as your:
+- [Kubernetes Ingress Controller](https://traefik.io/solutions/kubernetes-ingress/),
+- [Load Balancer](https://traefik.io/solutions/docker-swarm-ingress/),
+- [API Gateway](https://traefik.io/solutions/api-gateway/)
+
+Traefik Enterprise enables centralized access management, distributed Let's Encrypt, and other advanced capabilities. Learn more in [this 15-minute technical walkthrough](https://info.traefik.io/watch-traefikee-demo).
```
Comment (tomMoulard) [inline docs/content/getting-started/configuration-overview.md:104]:
```suggestion
    If you're using Traefik for commercial applications,
    consider the [Enterprise Edition](https://traefik.io/traefik-enterprise/).
    You can use it as your:

    - [Kubernetes Ingress Controller](https://traefik.io/solutions/kubernetes-ingress/),
    - [Load Balancer](https://traefik.io/solutions/docker-swarm-ingress/),
    - [API Gateway](https://traefik.io/solutions/api-gateway/)

    Traefik Enterprise enables centralized access management,
    distributed Let's Encrypt,
    and other advanced capabilities.
    Learn more in [this 15-minute technical walkthrough](https://info.traefik.io/watch-traefikee-demo).
```

This change goes from :

Screenshot from 2022-07-04 09-27-12

to:

Screenshot from 2022-07-04 09-27-22

Next time, feel free to test your changes on the documentation by doing `make docs-serve` :stuck_out_tongue:

File: docs/content/getting-started/quick-start.md [hunk 5]
```diff
@@ -114,3 +114,12 @@ IP: 172.27.0.4
...
     Now that you have a basic understanding of how Traefik can automatically create the routes to your services and load balance them, it is time to dive into [the documentation](/) and let Traefik work for you!
+
+!!! question "Using Traefik for Business Applications?"
+
+If you're using Traefik for commercial applications, consider the [Enterprise Edition](https://traefik.io/traefik-enterprise/). You can use it as your:
+- [Kubernetes Ingress Controller](https://traefik.io/solutions/kubernetes-ingress/),
+- [Load Balancer](https://traefik.io/solutions/docker-swarm-ingress/),
+- [API Gateway](https://traefik.io/solutions/api-gateway/)
+
+Traefik Enterprise enables centralized access management, distributed Let's Encrypt, and other advanced capabilities. Learn more in [this 15-minute technical walkthrough](https://info.traefik.io/watch-traefikee-demo).
```
Comment (tomMoulard) [inline docs/content/getting-started/quick-start.md:125]:
```suggestion
    If you're using Traefik for commercial applications,
    consider the [Enterprise Edition](https://traefik.io/traefik-enterprise/).
    You can use it as your:

    - [Kubernetes Ingress Controller](https://traefik.io/solutions/kubernetes-ingress/),
    - [Load Balancer](https://traefik.io/solutions/docker-swarm-ingress/),
    - [API Gateway](https://traefik.io/solutions/api-gateway/)

    Traefik Enterprise enables centralized access management,
    distributed Let's Encrypt,
    and other advanced capabilities.
    Learn more in [this 15-minute technical walkthrough](https://info.traefik.io/watch-traefikee-demo).
```

File: docs/content/https/acme.md [hunk 6]
```diff
@@ -666,3 +666,12 @@ If Let's Encrypt is not reachable, the following certificates will apply:
...
     For new (sub)domains which need Let's Encrypt authentication, the default Traefik certificate will be used until Traefik is restarted.
+    
+!!! question "Using Traefik for Business Applications?"
+
+If you're using Traefik for commercial applications, consider the [Enterprise Edition](https://traefik.io/traefik-enterprise/). You can use it as your:
+- [Kubernetes Ingress Controller](https://traefik.io/solutions/kubernetes-ingress/),
+- [Load Balancer](https://traefik.io/solutions/docker-swarm-ingress/),
+- [API Gateway](https://traefik.io/solutions/api-gateway/)
+
+Traefik Enterprise enables centralized access management, distributed Let's Encrypt, and other advanced capabilities. Learn more in [this 15-minute technical walkthrough](https://info.traefik.io/watch-traefikee-demo).
```
Comment (tomMoulard) [inline docs/content/https/acme.md:677]:
```suggestion
    If you're using Traefik for commercial applications,
    consider the [Enterprise Edition](https://traefik.io/traefik-enterprise/).
    You can use it as your:

    - [Kubernetes Ingress Controller](https://traefik.io/solutions/kubernetes-ingress/),
    - [Load Balancer](https://traefik.io/solutions/docker-swarm-ingress/),
    - [API Gateway](https://traefik.io/solutions/api-gateway/)

    Traefik Enterprise enables centralized access management,
    distributed Let's Encrypt,
    and other advanced capabilities.
    Learn more in [this 15-minute technical walkthrough](https://info.traefik.io/watch-traefikee-demo).
```

File: docs/content/providers/docker.md [hunk 7]
```diff
@@ -714,3 +714,12 @@ providers:
...
 ```
+
+!!! question "Using Traefik for Business Applications?"
+
+If you're using Traefik for commercial applications, consider the [Enterprise Edition](https://traefik.io/traefik-enterprise/). You can use it as your:
+- [Ingress Controller](https://traefik.io/solutions/kubernetes-ingress/),
+- [Load Balancer](https://traefik.io/solutions/docker-swarm-ingress/),
+- [API Gateway](https://traefik.io/solutions/api-gateway/)
+
+Traefik Enterprise enables centralized access management, distributed Let's Encrypt, and other advanced capabilities. Learn more in [this 15-minute technical walkthrough](https://info.traefik.io/watch-traefikee-demo).
```
Comment (tomMoulard) [inline docs/content/providers/docker.md:725]:
```suggestion
    If you're using Traefik for commercial applications,
    consider the [Enterprise Edition](https://traefik.io/traefik-enterprise/).
    You can use it as your:

    - [Kubernetes Ingress Controller](https://traefik.io/solutions/kubernetes-ingress/),
    - [Load Balancer](https://traefik.io/solutions/docker-swarm-ingress/),
    - [API Gateway](https://traefik.io/solutions/api-gateway/)

    Traefik Enterprise enables centralized access management,
    distributed Let's Encrypt,
    and other advanced capabilities.
    Learn more in [this 15-minute technical walkthrough](https://info.traefik.io/watch-traefikee-demo).
```

File: docs/content/providers/kubernetes-ingress.md [hunk 8]
```diff
@@ -501,3 +501,12 @@ providers:
...
 many examples of Ingresses definitions are located in the test [examples](https://github.com/traefik/traefik/tree/v2.8/pkg/provider/kubernetes/ingress/fixtures) of the Traefik repository.
+
+!!! question "Using Traefik for Business Applications?"
+
+If you're using Traefik for commercial applications, consider the [Enterprise Edition](https://traefik.io/traefik-enterprise/). You can use it as your:
+- [Kubernetes Ingress Controller](https://traefik.io/solutions/kubernetes-ingress/),
+- [Load Balancer](https://traefik.io/solutions/docker-swarm-ingress/),
+- [API Gateway](https://traefik.io/solutions/api-gateway/)
+
+Traefik Enterprise enables centralized access management, distributed Let's Encrypt, and other advanced capabilities. Learn more in [this 15-minute technical walkthrough](https://info.traefik.io/watch-traefikee-demo).
```
Comment (tomMoulard) [inline docs/content/providers/kubernetes-ingress.md:512]:
```suggestion
    If you're using Traefik for commercial applications,
    consider the [Enterprise Edition](https://traefik.io/traefik-enterprise/).
    You can use it as your:

    - [Kubernetes Ingress Controller](https://traefik.io/solutions/kubernetes-ingress/),
    - [Load Balancer](https://traefik.io/solutions/docker-swarm-ingress/),
    - [API Gateway](https://traefik.io/solutions/api-gateway/)

    Traefik Enterprise enables centralized access management,
    distributed Let's Encrypt,
    and other advanced capabilities.
    Learn more in [this 15-minute technical walkthrough](https://info.traefik.io/watch-traefikee-demo).
```

File: docs/content/routing/entrypoints.md [hunk 9]
```diff
@@ -967,3 +967,12 @@ entryPoints:
...
 ```
+
+!!! question "Using Traefik for Business Applications?"
+
+If you're using Traefik for commercial applications, consider the [Enterprise Edition](https://traefik.io/traefik-enterprise/). You can use it as your:
+- [Kubernetes Ingress Controller](https://traefik.io/solutions/kubernetes-ingress/),
+- [Load Balancer](https://traefik.io/solutions/docker-swarm-ingress/),
+- [API Gateway](https://traefik.io/solutions/api-gateway/)
+
+Traefik Enterprise enables centralized access management, distributed Let's Encrypt, and other advanced capabilities. Learn more in [this 15-minute technical walkthrough](https://info.traefik.io/watch-traefikee-demo).
```
Comment (tomMoulard) [inline docs/content/routing/entrypoints.md:978]:
```suggestion
    If you're using Traefik for commercial applications,
    consider the [Enterprise Edition](https://traefik.io/traefik-enterprise/).
    You can use it as your:

    - [Kubernetes Ingress Controller](https://traefik.io/solutions/kubernetes-ingress/),
    - [Load Balancer](https://traefik.io/solutions/docker-swarm-ingress/),
    - [API Gateway](https://traefik.io/solutions/api-gateway/)

    Traefik Enterprise enables centralized access management,
    distributed Let's Encrypt,
    and other advanced capabilities.
    Learn more in [this 15-minute technical walkthrough](https://info.traefik.io/watch-traefikee-demo).
```

File: docs/content/routing/routers/index.md [hunk 10]
```diff
@@ -1321,3 +1321,12 @@ There must be one (and only one) UDP [service](../services/index.md) referenced
...
 !!! important "UDP routers can only target UDP services (and not HTTP or TCP services)."
+
+!!! question "Using Traefik for Business Applications?"
+
+If you're using Traefik for commercial applications, consider the [Enterprise Edition](https://traefik.io/traefik-enterprise/). You can use it as your:
+- [Kubernetes Ingress Controller](https://traefik.io/solutions/kubernetes-ingress/),
+- [Load Balancer](https://traefik.io/solutions/docker-swarm-ingress/),
+- [API Gateway](https://traefik.io/solutions/api-gateway/)
+
+Traefik Enterprise enables centralized access management, distributed Let's Encrypt, and other advanced capabilities. Learn more in [this 15-minute technical walkthrough](https://info.traefik.io/watch-traefikee-demo).
```
Comment (tomMoulard) [inline docs/content/routing/routers/index.md:1332]:
```suggestion
    If you're using Traefik for commercial applications,
    consider the [Enterprise Edition](https://traefik.io/traefik-enterprise/).
    You can use it as your:

    - [Kubernetes Ingress Controller](https://traefik.io/solutions/kubernetes-ingress/),
    - [Load Balancer](https://traefik.io/solutions/docker-swarm-ingress/),
    - [API Gateway](https://traefik.io/solutions/api-gateway/)

    Traefik Enterprise enables centralized access management,
    distributed Let's Encrypt,
    and other advanced capabilities.
    Learn more in [this 15-minute technical walkthrough](https://info.traefik.io/watch-traefikee-demo).
```

File: docs/content/routing/services/index.md [hunk 11]
```diff
@@ -1645,3 +1645,12 @@ udp:
...
 ```
+
+!!! question "Using Traefik for Business Applications?"
+
+If you're using Traefik for commercial applications, consider the [Enterprise Edition](https://traefik.io/traefik-enterprise/). You can use it as your:
+- [Kubernetes Ingress Controller](https://traefik.io/solutions/kubernetes-ingress/),
+- [Load Balancer](https://traefik.io/solutions/docker-swarm-ingress/),
+- [API Gateway](https://traefik.io/solutions/api-gateway/)
+
+Traefik Enterprise enables centralized access management, distributed Let's Encrypt, and other advanced capabilities. Learn more in [this 15-minute technical walkthrough](https://info.traefik.io/watch-traefikee-demo).
```
Comment (tomMoulard) [inline docs/content/routing/services/index.md:1656]:
```suggestion
    If you're using Traefik for commercial applications,
    consider the [Enterprise Edition](https://traefik.io/traefik-enterprise/).
    You can use it as your:

    - [Kubernetes Ingress Controller](https://traefik.io/solutions/kubernetes-ingress/),
    - [Load Balancer](https://traefik.io/solutions/docker-swarm-ingress/),
    - [API Gateway](https://traefik.io/solutions/api-gateway/)

    Traefik Enterprise enables centralized access management,
    distributed Let's Encrypt,
    and other advanced capabilities.
    Learn more in [this 15-minute technical walkthrough](https://info.traefik.io/watch-traefikee-demo).
```

**Review by tomMoulard — APPROVED** [review]
LGTM :ok_hand:

**Review by kevinpollet — APPROVED** [review]
LGTM

**Review by ldez — APPROVED** [review]
LGTM

**Pull request merged**